operator/hack/common.sh

```diff
@@ -65,6 +65,26 @@ function retry() {
   return 1
 }
 
+function check_version_tag() {
+  local -r version_tag="$1"
+  local -r allow_dirty_tag="$2"
+
+  echo $IMAGE_TAG_BASE
+  if docker manifest inspect $IMAGE_TAG_BASE:$version_tag ; then
+    return 0
+  else
+    log "Cannot find $IMAGE_TAG_BASE:$version_tag image tag in the registry."
+  fi
+
+
+  if [[ $version_tag == *-dirty ]] && [[ $allow_dirty_tag = false ]]; then
+    log "Cannot install from *-dirty image tag. Please, use 'deploy-dirty-tag-via-olm' command instead if you need to install dirty tagged image"
```

Comment:
This message seems to be specific to one makefile target, but can be emitted in other contexts too.
Perhaps mentioning the variable would make more sense, but then someone might be invoking the scripts directly rather than via `make`, where the variable does not apply.

Reply:
You are right. I also added additional log just to explicitly note if it's dirty tagged image